packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -0,0 +1,380 @@
+                { icon: 'Forex', text: forex_label, highlighted: true },
+                { icon: 'Cryptocurrencies', text: localize('Cryptocurrencies'), highlighted: true },
+                { icon: 'ETF', text: localize('ETF'), highlighted: true },
+            ];
+    }
+};
+
+// Get the Account Title according to the market type and jurisdiction
+const getAccountCardTitle = (shortcode: string, is_demo?: boolean) => {
+    switch (shortcode) {
+        case 'synthetic_svg':
+            return is_demo ? localize('Derived Demo') : localize('Derived - SVG');
+        case 'synthetic_bvi':
+            return localize('Derived - BVI');
+        case 'synthetic_vanuatu':
+            return localize('Derived - Vanuatu');
+        case 'financial_svg':
+            return is_demo ? localize('Financial Demo') : localize('Financial - SVG');
+        case 'financial_bvi':
+            return localize('Financial - BVI');
+        case 'financial_vanuatu':
+            return localize('Financial - Vanuatu');
+        case 'financial_labuan':
+            return localize('Financial - Labuan');
+        case 'all_svg':
+            return is_demo ? localize('Swap-Free Demo') : localize('Swap-Free - SVG');
+        case 'dxtrade':
+            return is_demo ? localize('Deriv X Demo') : localize('Deriv X');
+        default:
+            return is_demo ? localize('CFDs Demo') : localize('CFDs');
+    }
+};
+
+// Get the Platform label
+const getPlatformLabel = (shortcode?: string) => {
+    switch (shortcode) {
+        case 'dxtrade':
+        case 'CFDs':
+            return 'Other CFDs';
+        case 'mt5':
+        default:
+            return 'MT5 Platform';
+    }
+};
+
+// Object to map the platform label
+const platfromsHeaderLabel = {
+    mt5: 'MT5 Platform',
+    other_cfds: 'Other CFDs',
+};
+
+// Get the Account Icons based on the market type
+const getAccountIcon = (shortcode: string) => {
+    switch (shortcode) {
+        case 'synthetic':
+            return 'Derived';
+        case 'financial':
+            return 'Financial';
+        case 'all':
+            return 'SwapFree';
+        case 'dxtrade':
+            return 'DerivX';
+        default:
+            return 'CFDs';
+    }
+};
+
+// Convert the market type from gaming to synthethics
+const getMarketType = (trading_platforms: TModifiedTradingPlatformAvailableAccount) => {
+    return trading_platforms.market_type === 'gaming' ? 'synthetic' : trading_platforms.market_type;
+};
+
+// Get the color of Header based on the platform
+const getHeaderColor = (shortcode: string) => {
+    switch (shortcode) {
+        case platfromsHeaderLabel.other_cfds:
+            return 'green';
+        case platfromsHeaderLabel.mt5:
+        default:
+            return 'blue';
+    }
+};
+
+// Config for different Jurisdictions
+const cfd_config = {
+    leverage: '1:1000',
+    leverage_description: localize('Maximum Leverage'),
```

Comment:
localize should be called in a function, otherwise, it won't be translated on language change.

Reply:
Quality team will come up with a linter to catch this soon.

Reply:
Missed that part of function
Thanks for pointing out.